.github/workflows/build_packages.yaml

```diff
@@ -269,32 +265,6 @@ jobs:
         path: .
     - name: unzip source code
       run: unzip -q source.zip
-    - name: downloads old emqx zip packages
-      env:
-        PROFILE: ${{ matrix.profile }}
-        ARCH: ${{ matrix.arch }}
-        SYSTEM: ${{ matrix.os }}
-        OLD_VSNS: ${{ needs.prepare.outputs.old_vsns }}
```

Comment:
old versions are used as a matrix dimension, doing relup tests in a loop would mean longer time?

Reply:
No, in older versions "old_vsn" was only used to pass values, not matrices